test/specs/visualforceLsp.e2e.ts

```diff
@@ -24,7 +24,7 @@ describe('Visualforce LSP', async () => {
 
     utilities.log(`${testSetup.testSuiteSuffixName} - calling createVisualforcePage()`);
     // Clear output before running the command
-    const workbench = await browser.getWorkbench();
+    const workbench = await (await browser.getWorkbench()).wait();
```

Comment:
Adding dynamic wait() for workbench helps. Don't have to estimate the wait time after each workbench call. Let's try and include it wherever we can and avoid hard pause. Good work 🎉

Reply:
It does! I'm adding that in my other PR 🙂